### DISCUSSION TITLE
Removing small grains after reconstruction

### DISCUSSION THREAD

**Discussion by huseyinhzl** [mtex-toolbox/mtex#1874] [Ask Anything]
Hi all,

I have been doing parent grain reconstruction in my titanium alloy. However, I had to remove some small grains that belong to the precipitate after parent grain reconstruction. I have tried several things, but could not manage to do that. Can anyone suggest how I can remove small grains after prior beta reconstruction or remove the precipitated phase from the reconstruction (I do not mean to make it notindexed at the begining)?

Thank you

**Comment by ralfHielscher** [comment]
if you have a list of grains `grains` then
```matlab
grains(grains.grainSize<5)=[];
```
will remove all grains from the list the have fewer then 5 pixels.
Ralf.
